docs/release_notes/2024/feb2024.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 1
+id: Feb_2023
+description: Release notes for February
+title: February 2024
+tags:
+  - release notes
+  - changelog
+---
+
+## 3.3.0 (Feb 05, 2024)
+
+- Prophecy Python libs version: 1.8.4
+- Prophecy Scala libs version: 7.1.66
+
+### Features {#FeaturesRelease330}
+
+#### Table Iterator Subgraph And Custom Subgraphs
+
+Introducing the Table Iterator Subgraph in Prophecy version 3.3. This new subgraph empowers users to iterate over one or more Gems for each row of the first input DataFrame. Additionally, users can create their own Custom Subgraphs to incorporate custom Iterators or control flow logic such as try-catch, conditional, etc.
+
+#### Automatic Code Regeneration
+
+Prophecy now facilitates automatic code regeneration whenever a user makes changes that could potentially impact the generated code for other Pipelines and Projects. For instance, when updating dependencies or reusable components like UDFs, Subgraph, Datasets, etc., the system automatically triggers updates to pipelines and jobs.
+This ensures seamless synchronization and alerts users to any potential impacts on their codebase.
+
+#### Secret Management
+
+Enhancing security, Prophecy now supports three different Secret Providers: Databricks Secrets, HashiCorp Vault, and Environment Variables. Users can link these providers in the Fabric they want to use and securely access secrets in their Gems and Pipelines. This ensures that sensitive authentication credentials are stored and used securely.
```

Comment:
> This ensures that sensitive authentication credentials are stored and used securely.
We do not store secret values in Prophecy. Can we please highlight it? This sentence conveys that we might store secrets in Prophecy, albeit in a secure way